roles/serviceassurance/tasks/_local_signing_authority.yml

```diff
@@ -7,77 +7,32 @@
       apiVersion: certmanager.k8s.io/v1alpha1
       kind: Issuer
       metadata:
-        name: '{{ meta.namespace }}-selfsigned'
+        name: '{{ meta.name }}-selfsigned'
```

Comment:
I guess I was thinking that having a per-cluster issuer made sense, but I admit that certificate management is a weak tool in my box. I don't have a strong justification for why I'm changing this other than gut instinct. Please guide me :)

Reply:
Guidance? Ummmm,  `if ! broken; do ! fix` ? :)

Reply:
:rotating_light: nerd alert! :rotating_light:

Reply:
Well since we only deploy one SAF per namespace, and we don't tend to deploy other things into the same namespace, then the name is basically equivalent either way. I don't care which as long as it works. If your gut likes this better, that's good enough for me!